### PR TITLE
Refactor the scapval test

### PR DESCRIPTION
**Pull request by matejak**
#### Description:

Make the code more explicit/readable. Successor of #10451

#### Rationale:

Nothing in particular, make the code readable and extensible.

#### Review Hints:

The functionality should be the same.

### CODE REVIEW

**Comment by matejak** [comment]
I propose to disregard CodeClimate suggestion, as the current layout allows us to easily add more conditions to the PASS case.

File: tests/run_scapval.py [hunk 1]
```diff
@@ -30,26 +37,39 @@ def parse_args():
...
 
+def ds_is_k8s_related(result_path):
+    for prodname in K8S_PRODUCTS:
+        ds_basename_stem = "ssg-{prodname}-ds".format(prodname=prodname)
+        if ds_basename_stem in result_path:
+            return True
+    return False
+
+
+def format_requirement_feedback(req_id, message):
```
Comment (jan-cerny) [inline tests/run_scapval.py:48]: I think it should be named print_requirement_feedback as its main goal is to print something, it doesn't return a formatted string.

**Review by jan-cerny — APPROVED** [review]

**Comment by marcusburghardt** [comment]
@jan-cerny , I believe it can already be merged, correct?

**Comment by jan-cerny** [comment]
@marcusburghardt Yes, it looks like the CI has already finished!

**Pull request merged**